command/node_status.go

```diff
@@ -111,6 +116,7 @@ func (c *NodeStatusCommand) Run(args []string) int {
 		fmt.Sprintf("Datacenter|%s", node.Datacenter),
 		fmt.Sprintf("Drain|%v", node.Drain),
 		fmt.Sprintf("Status|%s", node.Status),
+                fmt.Sprintf("Attributes|%s", strings.Join(attributes, ", ")),
```

Comment:
I think this is because that line uses spaces instead of tabs. `gofmt` should fix it. @achanda Not sure which editor you use but I have mine set to run `gofmt` when I save a file -- helps with things like this.